### DISCUSSION TITLE
求游戏搭子。

### DISCUSSION THREAD

**Discussion by lantainAirblue** [share121/inter-knot#4650] [委托]
一起玩最好聊得来的。
![Uploading Screenshot_绝区零_20240908_155436.jpg…]()

**Comment by tingyu919** [comment]
这个委托我接了（狗头）

**Comment by yzysrewrwqr** [comment]
这个委托是限女生吧<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by lllliufeng** [comment]
这个委托我不接了
